### ISSUE TITLE
[kws_streaming] How to realize BC-ResNet running in streaming mode

### ISSUE THREAD

**Issue by panchengww** [google-research/google-research#1158]
@rybakov 
Hi,
I am preparing to use BC-ResNet to implement streaming inference. When command below used(padding is same), it seems unable to stream inference. 
If padding is set to causal, dimension inconsistency will occur in streaming inference。
Whether `kws_streaming/models/bc_resnet.py` needs to be modified to enable streaming inference?

Thank you

```
$CMD_TRAIN \
--data_url '' \
--data_dir $DATA_PATH/ \
--train_dir $MODELS_PATH/bc_resnet_1/ \
--mel_upper_edge_hertz 7500 \
--mel_lower_edge_hertz 125 \
--how_many_training_steps 100,100,100,100,30000,30000,20000,10000,5000,5000 \
--learning_rate 0.001,0.002,0.003,0.004,0.005,0.002,0.0005,1e-5,1e-6,1e-7 \
--window_size_ms 30.0 \
--window_stride_ms 10.0 \
--mel_num_bins 40 \
--dct_num_features 0 \
--resample 0.1 \
--alsologtostderr \
--train 1 \
--use_spec_augment 0 \
--time_masks_number 2 \
--time_mask_max_size 25 \
--frequency_masks_number 2 \
--frequency_mask_max_size 7 \
--pick_deterministically 1 \
bc_resnet \
--sub_groups 5 \
--last_filters 32 \
--first_filters 16 \
--paddings 'same' \
--dilations '(1,1),(2,1),(4,1),(8,1)' \
--strides '(1,1),(1,2),(1,2),(1,1)' \
--blocks_n '2, 2, 4, 4' \
--filters '8, 12, 16, 20' \
--dropouts '0.1, 0.1, 0.1, 0.1' \
--pools '1, 1, 1, 1' \
--max_pool 0
```

> tflite Final test accuracy, non stream model = 96.50% (N=4890)
> tf stream model state external with reset_state 1
> TF Final test accuracy of stream model state external = 0.00% (N=10)
> tf stream model state external with reset_state 0
> TF Final test accuracy of stream model state external = 0.00% (N=10)

**Comment by rybakov** [comment]
Hi @panchengww, the evaluation tool for model streaming was designed for non strided models (and was not extended, please feel free to update it). I would suggest to run the evaluation with [colab](https://github.com/google-research/google-research/blob/master/kws_streaming/colab/02_inference.ipynb)
